applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.c

```diff
@@ -202,9 +202,17 @@ static int adv_create(void)
 	NET_BUF_SIMPLE_DEFINE_STATIC(brdcst_id_buf, BT_UUID_SIZE_16 + BT_AUDIO_BROADCAST_ID_SIZE);
 	/* Buffer for Appearance */
 	NET_BUF_SIMPLE_DEFINE_STATIC(brdcst_appearance_buf, BT_UUID_SIZE_16);
+	/* Buffer for manufacturer ID */
+	NET_BUF_SIMPLE_DEFINE_STATIC(manufacturer_id_buf, BT_UUID_SIZE_16);
 	/* Buffer for Public Broadcast Announcement */
 	NET_BUF_SIMPLE_DEFINE_STATIC(base_buf, 128);
 
+	net_buf_simple_add_le16(&manufacturer_id_buf, CONFIG_BT_DEVICE_MANUFACTURER_ID);
```

Comment:
I don't quite see why this is added directly here, whilst there is a function created to do the same thing in another location.

Reply:
Created OCT-2949 to get these more similar